tf2onnx/tf_loader.py

```diff
@@ -684,8 +684,11 @@ def tf_optimize_grappler(input_names, output_names, graph_def, fold_constant=Non
     # depends on so for now don't turn this on, fold_constant is always enabled now.
     rewrite_options.optimizers[:] = [
         # 'pruning', 'constfold', 'arithmetic', 'dependency', 'function',
-        'constfold', 'function'
+        'function', 'dependency'
     ]
+    # This flag disables folding QDQ nodes around constants in the network (eg: around conv/FC weights)
+    rewrite_options.experimental_disable_folding_quantization_emulation = True
```

Comment:
This does not exist in tensorflow v1.x versions which is also need to be supported by us.

Please add a validation(is_tf2()) here to make sure this will only be called when tf version is 2.x so the CI could pass.

Reply:
Thank you @fatcat-z Added it now.